app/src/main/res/values/strings.xml

```diff
@@ -65,6 +65,7 @@
     <string name="storage_fake_drive_summary">Needs to be plugged in</string>
     <string name="storage_available_bytes"><xliff:g example="1 GB" id="size">%1$s</xliff:g> free</string>
     <string name="storage_not_recommended"><xliff:g example="Skynet">%1$s</xliff:g> (Not recommended)</string>
+    <string name="storage_round_sync_summary_prefix" translatable="false">RoundSync Remote: </string>
```

Comment:
Are we sure roundsync isn't translated into other languages? What about `Remote:`?